akka-actor/src/main/scala/akka/routing/ConsistentHashing.scala

```diff
@@ -5,18 +5,18 @@
 package akka.routing
 
 import java.util.concurrent.atomic.AtomicReference
-
 import scala.collection.immutable
 import scala.util.control.NonFatal
-
 import com.typesafe.config.Config
-
-import akka.actor.ActorPath
-import akka.actor.ActorSystem
-import akka.actor.Address
-import akka.actor.ExtendedActorSystem
-import akka.actor.SupervisorStrategy
-import akka.actor.WrappedMessage
+import akka.actor.{
+  ActorPath,
+  ActorSystem,
+  Address,
+  ExtendedActorSystem,
+  InvalidMessageException,
+  SupervisorStrategy,
+  WrappedMessage
+}
```

Comment:
We have in general tried to go with individual import lines rather than import groups like this since it makes merge conflicts because of imports somewhat less common.

Reply:
Seems IDEA did this.will revert